src/main.js

```diff
@@ -699,14 +700,64 @@ function getNodePosition(node){
   return d3.transform(d3.select(node).attr("transform")).translate;
 }
 
+
 function addEleContent(e) {
-  var newNodeAddress = "http://webstrates.ucsd.edu/" + hoveredEle;
+  let newNodeAddress = "http://webstrates.ucsd.edu/" + hoveredEle;
+  let wrapper = document.createElement('div');
+  wrapper.setAttribute("id", "addContentWrapper");
+  
+  let toFrame = '<iframe id="addWindow" src="'+ newNodeAddress + '" style="width: 95%; height: 350px;"><p>ERROR: Your browser does not support iframes.</p></iframe>';
+  let addNoteButton = '<button type="button" id="addNoteBtn" onclick="appendNote()">Add Sticky Note</button> ';
+  let addPicButton = '<input type="file" id="addPicBtn" onchange="appendPic()"> ';
+  let closeButton = '<button type="button" id="closeWindowBtn" onclick="closeContentWindow()">Close Content Window</button> ';
+  wrapper.innerHTML = toFrame + addNoteButton + addPicButton + closeButton;
+  document.getElementById("content_container").appendChild(wrapper);
+
+  addEleToList(e);
+    
+  addWindowOpen = true;
+
+}
+
+function closeContentWindow() {
+  let wrapper = document.getElementById("addContentWrapper");
+  let childOne = document.getElementById("addWindow");
```

Comment:
Better to change "childOne" to "addWindowFrame".

Reply:
Done